CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * Your contribution here.
 
 #### Fixes
+
+* [#1995](https://github.com/ruby-grape/grape/pull/1995): Address some Ruby warnings (undefined instance variables, method redefined) - [@nbeyer](https://github.com/nbeyer).
```

Comment:
Or just `Fix: "undefined instance variables" and "method redefined" warnings`?

Reply:
Sure. Changed at 4fd9c2c8048163a0cfdb3036e4c50c5629e08e28